cypress/e2e/pages/homePage/index.cy.js

```diff
@@ -63,9 +171,37 @@ const testSuites = [
   },
   {
     path: '/uzbek/cyr',
-    runforEnv: ['local', 'test', 'live'],
-    service: '/uzbek/cyr',
-    tests,
+    runforEnv: ['test', 'live'],
+    service: 'uzbek',
+    pageIdentifier: 'uzbekcyr.page',
+    siteId: 96,
+    applicationType: 'responsive',
+    contentType: 'index-home',
+    useReverb: true,
+    tests: [
+      ...tests,
+      assertPageView,
+      assertMessageBannerComponentView,
+      assertMessageBannerComponentClick,
+      assertMostReadComponentView,
+      assertMostReadComponentClick,
+    ],
+  },
+  // analytics test only
+  {
+    path: '/afrique',
+    runforEnv: ['local', 'test'],
+    service: 'afrique',
+    pageIdentifier: 'afrique.page',
+    siteId: 3,
+    applicationType: 'responsive',
+    contentType: 'index-home',
+    useReverb: true,
+    tests: [
+      assertPageView,
+      assertBillboardComponentView,
+      assertBillboardComponentClick,
```

Comment:
If we have a test for billboard on local & test for another service, we might not need this here.